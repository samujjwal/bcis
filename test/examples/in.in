unit inch
unit foot = 12*inch
print(36*inch in foot)
