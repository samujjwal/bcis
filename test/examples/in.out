3.0*foot
